projects/ion/src/lib/icon/svgs/icons.ts

```diff
@@ -1,21 +1,56 @@
+import {
```

Comment:
very good! could you create the calendar icon?

Reply:
Yes.

Reply:
But in the future there will be a feature to add the others.

Reply:
Done.